Auth0Tests/AuthenticationSpec.swift

```diff
@@ -552,6 +556,28 @@ class AuthenticationSpec: QuickSpec {
                             }
                     }
                 }
+                
+                it("should request passkey signup challenge with organization and default parameters") {
```

Comment:
There is no need for a new test case here. Like for the previous methods, updating the "all parameters" test case should suffice. E.g. the "should request passkey signup challenge with all parameters" one.